package.json

```diff
@@ -27,16 +27,12 @@
     "preversion": "npm run docs:build && git add ./docs",
     "report:coveralls": "istanbul cover _mocha --root lib --include-all-sources --report lcovonly --dir coveralls -- ./test/index.js && cat ./coveralls/lcov.info | coveralls && rm -rf ./coveralls",
     "report:coverage": "istanbul cover _mocha --root lib --include-all-sources --report html --dir coverage-report -- ./test/index.js",
-    "report:complexity:create-config": "yaml2json .eslintrc.yml > .eslintrc",
```

Comment:
I think this means we can get rid of the `yamljs` dev dependency.

Reply:
Indeed, just removed it.